# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1546980140 -3600
#      Tue Jan 08 21:42:20 2019 +0100
# Branch stable
# Node ID 8beca28b7eb5de167a63cf58ae6b17162ec7d8af
# Parent  6e0bf8212b40b5a94381d8e2deee573197139340
* pt_BR.ts: updated translation by Felipe G. Nievinski

diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_BR.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_BR.ts
@@ -97,17 +97,17 @@
     <message>
         <location line="+5"/>
         <source>Select All</source>
         <translation>Selecionar Tudo</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Run Selection</source>
-        <translation type="unfinished">Executar Seleção</translation>
+        <translation>Executar Seleção</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Clear Window</source>
         <translation>Limpar Tudo</translation>
     </message>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+49"/>
@@ -383,74 +383,76 @@ file. Search capabilities may be affecte
 temporário. A funcionalidade de busca pode estar afetada.
 %1</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Could not setup the data required for the
 documentation viewer. Only help texts in
 the Command Window will be available.</source>
-        <translation type="unfinished"></translation>
+        <translation>Não foi possível configurar os dados necessários
+para o visualizador de documentação. Apenas textos
+de ajuda na Janela de Comando estarão disponíveis.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Find:</source>
-        <translation>Buscar:</translation>
+        <translation>Localizar:</translation>
     </message>
     <message>
         <location line="+7"/>
         <location line="+1"/>
         <source>Search forward</source>
         <translation>Buscar adiante</translation>
     </message>
     <message>
         <location line="+5"/>
         <location line="+1"/>
         <source>Search backward</source>
         <translation>Buscar para trás</translation>
     </message>
     <message>
         <location line="+74"/>
         <source>Function Index</source>
-        <translation type="unfinished"></translation>
+        <translation>Índice de Funções</translation>
     </message>
     <message>
         <location line="+98"/>
         <source>Go home</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir para o início</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Go back</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir para trás</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Go forward</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir para frente</translation>
     </message>
     <message>
         <location line="+43"/>
         <source>Find</source>
-        <translation type="unfinished">Localizar</translation>
+        <translation>Localizar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Zoom in</source>
-        <translation type="unfinished"></translation>
+        <translation>Mais zoom</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Zoom out</source>
-        <translation type="unfinished"></translation>
+        <translation>Menos zoom</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Zoom original</source>
-        <translation type="unfinished"></translation>
+        <translation>Zoom original</translation>
     </message>
     <message>
         <location line="-206"/>
         <source>Contents</source>
         <translation>Conteúdo</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -461,22 +463,22 @@ the Command Window will be available.</s
         <location line="+10"/>
         <location line="+39"/>
         <source>Search</source>
         <translation>Buscar</translation>
     </message>
     <message>
         <location line="+84"/>
         <source>Previous pages</source>
-        <translation type="unfinished"></translation>
+        <translation>Páginas anteriores</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Next pages</source>
-        <translation type="unfinished"></translation>
+        <translation>Páginas seguintes</translation>
     </message>
     <message>
         <location line="+383"/>
         <source>Unable to register help file %1.</source>
         <translation>Não foi possível registrar o arquivo de ajuda %1.</translation>
     </message>
 </context>
 <context>
@@ -492,37 +494,37 @@ the Command Window will be available.</s
         <translation>Veja a documentação para obter ajuda.</translation>
     </message>
 </context>
 <context>
     <name>octave::dw_main_window</name>
     <message>
         <location filename="../src/dw-main-window.cc" line="+49"/>
         <source>&amp;Close</source>
-        <translation type="unfinished">&amp;Fechar</translation>
+        <translation>&amp;Fechar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Close &amp;All</source>
-        <translation type="unfinished"></translation>
+        <translation>Fechar &amp;Todos</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Close &amp;Other</source>
-        <translation type="unfinished"></translation>
+        <translation>Fechar as &amp;Demais Abas</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Switch to &amp;Left Widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Alternar para Aba à &amp;Esquerda</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Switch to &amp;Right Widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Alternar para Aba à &amp;Direita</translation>
     </message>
 </context>
 <context>
     <name>octave::external_editor_interface</name>
     <message>
         <location filename="../src/external-editor-interface.cc" line="+63"/>
         <location line="+45"/>
         <source>Octave Editor</source>
@@ -797,17 +799,17 @@ para escrita: %2.</translation>
     <message>
         <location line="+3"/>
         <source>Navi&amp;gation</source>
         <translation>Nave&amp;gação</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Go &amp;to Line...</source>
-        <translation>Vá para &amp;Linha...</translation>
+        <translation>Ir para &amp;Linha...</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Move to Matching Brace</source>
         <translation>Mover para Parêntese/Chave/Colchete Correspondente</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1014,22 +1016,23 @@ para escrita: %2.</translation>
         <location line="+36"/>
         <source>Octave Editor</source>
         <translation>Editor do Octave</translation>
     </message>
     <message>
         <location line="-1977"/>
         <source>Cannot add breakpoint to modified or unnamed file.
 Save and add breakpoint, or cancel?</source>
-        <translation type="unfinished"></translation>
+        <translation>Não é possível adicionar o ponto de parada a arquivo modificado ou sem nome.
+Salvar e adicionar ponto de parada ou cancelar?</translation>
     </message>
     <message>
         <location line="+833"/>
         <source>Goto line</source>
-        <translation>Vá para linha</translation>
+        <translation>Ir para linha</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Line number</source>
         <translation>Número da linha</translation>
     </message>
     <message>
         <location line="+126"/>
@@ -1069,48 +1072,56 @@ está prestes a ser fechado mas foi modificado.  %2</translation>
     </message>
     <message>
         <location line="+126"/>
         <source>There were problems reading the file
 %1
 with the selected encoding %2.
 
 Modifying and saving the file might cause data loss!</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocorreram problemas ao ler o arquivo
+%1
+com a codificação selecionada %2.
+
+Modificar e salvar o arquivo pode causar perda de dados!</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Edit anyway</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Editar mesmo assim</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+57"/>
         <source>Chan&amp;ge encoding</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Mudar codificação</translation>
     </message>
     <message>
         <location line="-56"/>
         <location line="+49"/>
         <source>&amp;Close</source>
-        <translation type="unfinished">&amp;Fechar</translation>
+        <translation>&amp;Fechar</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Select new default encoding</source>
-        <translation type="unfinished"></translation>
+        <translation>Selecionar nova codificação padrão</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Please select a new encoding
 for reloading the current file.
 
 This does not change the default encoding.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Favor selecionar uma nova codificação
+para recarregar o arquivo atual.
+
+Isso não altera a codificação padrão.
+</translation>
     </message>
     <message>
         <location line="+212"/>
         <source>Debug or Save</source>
         <translation>Depurar ou Salvar</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1127,26 +1138,33 @@ Abandonar depuração e salvar?</translation>
 %2.</translation>
     </message>
     <message>
         <location line="+182"/>
         <source>The current encoding %1
 can not be applied.
 
 Please select another one!</source>
-        <translation type="unfinished"></translation>
+        <translation>A codificação atual %1
+não pode ser aplicada.
+
+Favor selecionar outra!</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
 with the selected encoding %1.
 Using it would result in data loss!
 
 Please select another one!</source>
-        <translation type="unfinished"></translation>
+        <translation>O conteúdo atual do editor não pode ser codificado
+usando a codificação selecionada %1.
+Usá-la resultaria em perda de dados!
+
+Favor selecionar outra!</translation>
     </message>
     <message>
         <location line="-115"/>
         <source>Octave Files (*.m)</source>
         <translation>Arquivos do Octave (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1264,17 +1282,17 @@ parece ter sido deletado ou renomeado. Gostaria de salvá-lo agora?%2</translation>
     <message>
         <location line="+4"/>
         <source>Actions on current directory</source>
         <translation>Ações no diretório atual</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Home Directory</source>
-        <translation>Exibir Diretório Pessoal</translation>
+        <translation>Exibir Diretório Inicial</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Set Browser Directory...</source>
         <translation>Definir Diretório a Navegar...</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -1292,17 +1310,17 @@ parece ter sido deletado ou renomeado. Gostaria de salvá-lo agora?%2</translation>
         <location line="-381"/>
         <location line="+384"/>
         <source>New Directory...</source>
         <translation>Novo Diretório...</translation>
     </message>
     <message>
         <location line="-329"/>
         <source>Double-click to open file/folder, right click for alternatives</source>
-        <translation type="unfinished"></translation>
+        <translation>Clique-duplo para abrir arquivo/pasta, botão direito para alternativas</translation>
     </message>
     <message>
         <location line="+282"/>
         <source>Open</source>
         <translation>Abrir</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1785,17 +1803,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Show filter</source>
         <translation>Exibir filtro</translation>
     </message>
     <message>
         <location line="+121"/>
         <source>Double-click a command to transfer it to the Command Window.</source>
-        <translation type="unfinished"></translation>
+        <translation>Dê clique-duplo sobre um comando para transferí-lo à Janela de Comandos.</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Enter text to filter the command history</source>
         <translation>Digite para filtrar o histórico de comandos</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1922,17 +1940,18 @@ Clique em &apos;Avançar&apos; para criar um arquivo de configuração e iniciar o Octave.&lt;/p&gt;
 </source>
         <translation>Nome da nova função:
 </translation>
     </message>
     <message>
         <location line="+44"/>
         <source>%1 is a built-in, compiled or inline
 function and can not be edited.</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 é uma função embutida, compilada ou inline
+e não pode ser editada.</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Can not find function %1</source>
         <translation>Não foi possível encontrar a função %1</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -2419,64 +2438,64 @@ A interface gráfica do Octave precisa ser fechada agora.</translation>
         <translation>Erro Crítico no Octave</translation>
     </message>
 </context>
 <context>
     <name>octave::settings_dialog</name>
     <message>
         <location filename="../src/settings-dialog.cc" line="-573"/>
         <source>Octave Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Preferências do Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save preferences.  Missing preferences file or unknown directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Não é possível salvar as preferências. Arquivo de preferências ausente ou diretório desconhecido.</translation>
     </message>
     <message>
         <location line="+15"/>
         <location line="+4"/>
         <location line="+700"/>
         <source>System setting</source>
-        <translation type="unfinished">Configuração do sistema</translation>
+        <translation>Configuração do sistema</translation>
     </message>
     <message>
         <location line="-448"/>
         <source>IBeam Cursor</source>
-        <translation type="unfinished">Cursor IBeam</translation>
+        <translation>Cursor &quot;I&quot;</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
-        <translation type="unfinished">Cursor de Bloco</translation>
+        <translation>Cursor de Bloco</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
-        <translation type="unfinished">Cursor Sublinhado</translation>
+        <translation>Cursor Sublinhado</translation>
     </message>
     <message>
         <location line="+164"/>
         <source>Set Octave Startup Directory</source>
-        <translation type="unfinished">Definir Diretório de Inicialização do Octave</translation>
+        <translation>Definir Diretório de Inicialização do Octave</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set File Browser Startup Directory</source>
-        <translation type="unfinished">Alterar Diretório de Inicialização do Navegador de Arquivos</translation>
+        <translation>Definir Diretório de Inicialização do Navegador de Arquivos</translation>
     </message>
     <message>
         <location line="+503"/>
         <source>Enable attribute colors</source>
-        <translation type="unfinished"></translation>
+        <translation>Habilitar cores para atributos</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Hide tools tips</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocultar dicas contextuais</translation>
     </message>
 </context>
 <context>
     <name>octave::setup_community_news</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+42"/>
         <source>Community News</source>
         <translation>Novidades da Comunidade</translation>
@@ -2966,17 +2985,17 @@ Você deseja usar a tecla de atalho mesmo assim, removendo-a da outra ação?</translation>
     <message>
         <location line="+2"/>
         <source>Convert Line Endings to Mac</source>
         <translation>Converter Finais de Linha para Mac</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Goto Line</source>
-        <translation>Vá para Linha</translation>
+        <translation>Ir para Linha</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move to Matching Brace</source>
         <translation>Mover para Parêntese/Chave/Colchete Correspondente</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3046,37 +3065,37 @@ Você deseja usar a tecla de atalho mesmo assim, removendo-a da outra ação?</translation>
     <message>
         <location line="+2"/>
         <source>Show Horizontal Scrollbar</source>
         <translation>Exibir Barra de Rolagem Horizontal</translation>
     </message>
     <message>
         <location line="+154"/>
         <source>Documentation Viewer</source>
-        <translation type="unfinished"></translation>
+        <translation>Visualizador de Documentação</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Navegador</translation>
     </message>
     <message>
         <location line="+79"/>
         <source>Import shortcuts from file...</source>
-        <translation type="unfinished"></translation>
+        <translation>Importar teclas de atalho do arquivo...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Export shortcuts to file...</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportar teclas de atalho para arquivo...</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Failed to open %1 as Octave shortcut file</source>
-        <translation type="unfinished"></translation>
+        <translation>Falha ao abrir %1 como arquivo de teclas de atalho do Octave</translation>
     </message>
     <message>
         <location line="-367"/>
         <source>Zoom In</source>
         <translation>Mais Zoom</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -3127,27 +3146,27 @@ Você deseja usar a tecla de atalho mesmo assim, removendo-a da outra ação?</translation>
     <message>
         <location line="+2"/>
         <source>Document on Keyword</source>
         <translation>Documentação sobre Palavra-chave</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Go to Homepage</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir para página inicial</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Go Back one Page</source>
-        <translation type="unfinished"></translation>
+        <translation>Voltar uma Página</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Go Forward one Page</source>
-        <translation type="unfinished"></translation>
+        <translation>Avançar uma Página</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Global</source>
         <translation>Global</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -3186,22 +3205,22 @@ Você deseja usar a tecla de atalho mesmo assim, removendo-a da outra ação?</translation>
     <message>
         <location line="+2"/>
         <source>Tab Handling in Dock Widgets</source>
         <translation>Manuseio de Abas no Acoplamento de Paineis</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find &amp; Replace in Dock Widgets</source>
-        <translation type="unfinished"></translation>
+        <translation>Localizar e Substituir em Painéis</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zooming in Editor and Documentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Zoom no Editor e na Documentação</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>View Menu</source>
         <translation>Menu Exibir</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -3876,77 +3895,77 @@ Gostaria de salvar o conjunto atual de teclas de atalho ou cancelar a ação?</translation>
     <message>
         <location line="-962"/>
         <source>Show horizontal scroll bar</source>
         <translation>Exibir barra de rolagem horizontal</translation>
     </message>
     <message>
         <location line="-1037"/>
         <source>Preferences</source>
-        <translation type="unfinished">Preferências</translation>
+        <translation>Preferências</translation>
     </message>
     <message>
         <location line="+57"/>
         <source>(requires restart)</source>
-        <translation type="unfinished"></translation>
+        <translation>(requer reinicialização)</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Use native file dialogs</source>
-        <translation type="unfinished"></translation>
+        <translation>Usar caixas de diálogo de arquivos nativas</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Use system icon theme if available (requires restart)</source>
-        <translation type="unfinished"></translation>
+        <translation>Use o tema de ícones do sistema, se disponível (requer reinicialização)</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>Toolbar Icons</source>
-        <translation type="unfinished"></translation>
+        <translation>Ícones da Barra de Ferramentas</translation>
     </message>
     <message>
         <location line="+186"/>
         <source>Language</source>
-        <translation type="unfinished"></translation>
+        <translation>Idioma</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Style</source>
-        <translation type="unfinished"></translation>
+        <translation>Estilo</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Initial working directory of Octave interpreter</source>
-        <translation type="unfinished"></translation>
+        <translation>Diretório de trabalho inicial do interpretador do Octave</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Restore last working directory of previous session</source>
-        <translation type="unfinished"></translation>
+        <translation>Restaurar último diretório de trabalho da sessão anterior</translation>
     </message>
     <message>
         <location line="+45"/>
         <source>Command</source>
-        <translation type="unfinished"></translation>
+        <translation>Comando</translation>
     </message>
     <message>
         <location line="+208"/>
         <source>Set focus to Command Window when running a command from within another widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Definir o foco na Janela de Comandos ao executar um comando de outro painel</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Print debug location in Command Window in addition to the marker in the editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Imprimir localização de depuração na Janela de Comandos, além do marcador no editor</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>Command Window Colors</source>
-        <translation type="unfinished"></translation>
+        <translation>Cores da Janela de Comandos</translation>
     </message>
     <message>
         <location line="+172"/>
         <source>Show complete path in title</source>
         <translation>Exibir caminho completo no título</translation>
     </message>
     <message>
         <location line="+100"/>
@@ -4061,22 +4080,22 @@ Gostaria de salvar o conjunto atual de teclas de atalho ou cancelar a ação?</translation>
     <message>
         <location line="+64"/>
         <source>Show completion list automatically</source>
         <translation>Exibir automaticamente lista de auto-completar</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Debugging</source>
-        <translation type="unfinished"></translation>
+        <translation>Depuração</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Always show debug breakpoints and pointers (opens related file if closed)</source>
-        <translation type="unfinished"></translation>
+        <translation>Sempre exibir apontadores e pontos de parada na depuração (abrundo arquivos relacionados se estiverem fechados)</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>File handling</source>
         <translation>Manipulação de arquivos</translation>
     </message>
     <message>
         <location line="+19"/>
@@ -4121,17 +4140,17 @@ Gostaria de salvar o conjunto atual de teclas de atalho ou cancelar a ação?</translation>
     <message>
         <location line="+22"/>
         <source>Restore last directory of previous session</source>
         <translation>Restaurar diretório da sessão anterior</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Initial file browser directory (only if not synchronized with initial working directory of Octave)</source>
-        <translation type="unfinished"></translation>
+        <translation>Diretório inicial do navegador de arquivos (somente se não estiver sincronizado com o diretório de trabalho inicial do Octave)</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Extensions of files to be opened in the default text editor (separated by &quot;;&quot;):</source>
         <translation>Extensões de arquivos a serem abertas no editor de texto padrão (separadas por &quot;;&quot;):</translation>
     </message>
     <message>
         <location line="+39"/>
@@ -4141,17 +4160,17 @@ Gostaria de salvar o conjunto atual de teclas de atalho ou cancelar a ação?</translation>
     <message>
         <location line="+35"/>
         <source>Colors for variable attributes</source>
         <translation>Cores para atributos de variáveis</translation>
     </message>
     <message>
         <location line="+107"/>
         <source>Use Command Window font</source>
-        <translation type="unfinished"></translation>
+        <translation>Usar a fonte da Janela de Comandos</translation>
     </message>
     <message>
         <location line="+196"/>
         <source>Import shortcut set</source>
         <translation>Importar conjunto de teclas de atalho</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -4237,17 +4256,17 @@ Gostaria de salvar o conjunto atual de teclas de atalho ou cancelar a ação?</translation>
     <message>
         <location line="+12"/>
         <source>Use alternating row colors</source>
         <translation>Usar cores de linha alternadas</translation>
     </message>
     <message>
         <location line="+59"/>
         <source>Disable global shortcuts when Command Window has focus</source>
-        <translation type="unfinished"></translation>
+        <translation>Desativar teclas de atalho globais quando a Janela de Comando tiver foco</translation>
     </message>
     <message>
         <location line="+211"/>
         <source>HttpProxy</source>
         <translation>Proxy HTTP</translation>
     </message>
     <message>
         <location line="-2559"/>
